### ISSUE TITLE
Ability to send deaths/achievements to different channels besides the active one

### ISSUE THREAD

**Issue by drinfernoo** [duke605/DiscordCE#3]
Would this be possible? Like if I have #general for normal chat, but want achievement announcements to go to #achievements?

**Comment by duke605** [comment]
Sure I can build that in

**Comment by drinfernoo** [comment]
I'm also not seeing achievements getting sent if the mod is running on a server, but I'm not sure if that's intended or not.

**Comment by drinfernoo** [comment]
I may have just missed something during my server setup.

**Comment by duke605** [comment]
Unfortunately the achievement code is called server side and my code only runs client side (hence the name). When you're playing single player the server is integrated meaning it also runs client side so it will run the achievement code.

**Comment by drinfernoo** [comment]
Would it be possible to make that code run on the server side when the mod is installed on a server?

**Comment by duke605** [comment]
No. The mod is only meant to be installed on the client. I made is purposefully crashes if installed on a server because it just won't work with the GUIs
